androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/splash/SplashScreen.kt

```diff
@@ -1,78 +1,240 @@
             Image(
-                painter = painterResource(id = R.drawable.logo_white),
+                painter = painterResource(id = R.drawable.bg_splash),
                 contentDescription = null,
-                contentScale = ContentScale.Crop
+                contentScale = ContentScale.Crop,
+                modifier = Modifier
+                    .matchParentSize()
+                    .blur(animateBlur.dp)
+            )
+
+            val gradient = Brush.verticalGradient(
+                colors = listOf(
+                    Black.copy(alpha = animateAlpha * InitialTopGradientAlpha),
+                    Black.copy(
+                        alpha = (animateAlpha * BottomGradientAlphaMultiplier) +
+                            InitialBottomGradientAlpha
+                    )
+                )
+            )
+            Box(
+                modifier = Modifier
+                    .matchParentSize()
+                    .background(gradient)
             )
+
+            AnimatedVisibility(
+                visible = shouldShowLogo,
+                enter = fadeIn(animationSpec = tween(LogoDurationInMillis))
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.logo_white),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .offset(animateLogoOffset.x.dp, animateLogoOffset.y.dp)
+                        .scale(animateLogoScale)
+                )
+            }
         }
     }
 }
 
+@Composable
+private fun LoginForm(
+    onLogInClick: (email: String, password: String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    Column(
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        modifier = modifier
+            .fillMaxSize()
+            .wrapContentHeight()
+            .padding(horizontal = 24.dp)
+    ) {
+        PrimaryTextField(
+            value = email,
+            onValueChange = { email = it },
+            placeholder = stringResource(id = R.string.login_email),
+            keyboardType = KeyboardType.Email,
+        )
+        Box {
+            PrimaryTextField(
+                value = password,
+                onValueChange = { password = it },
+                placeholder = stringResource(id = R.string.login_password),
+                visualTransformation = PasswordVisualTransformation(),
+                imeAction = ImeAction.Done,
+            )
+            Text(
+                text = stringResource(id = R.string.login_forgot),
+                color = MaterialTheme.colors.onSurface.copy(alpha = ForgotTextAlpha),
+                style = MaterialTheme.typography.body2,
+                modifier = Modifier
+                    .wrapContentHeight()
+                    .align(Alignment.CenterEnd)
+                    .padding(end = 18.dp)
+            )
+        }
```

Comment:
This will cause ui bug if the password is long enough to fill the text field. We can use `trailingIcon` of `TextField` to avoid overflowing.
<img src="https://user-images.githubusercontent.com/32578035/235600159-25f548ba-b231-46d6-9ef4-52e8c3095a60.png" width="360" height="800">